### PR TITLE
AKU-830 support for Alfresco v1 Upload API (see RA-637)

### PR DESCRIPTION
**Pull request by kevinroast**

### CODE REVIEW

**Comment by draperd** [comment]
This PR has been previously discussed with Kevin... no need to merge, we're going to check out the branch and build in the tests for this requirement based on the code provided in this PR.

**Comment by kevinroast** [comment]
:thumbsup:

File: aikau/src/main/resources/alfresco/services/_BaseUploadService.js [hunk 1]
```diff
@@ -437,17 +449,32 @@ define(["alfresco/core/CoreXhr",
...
+                  fileInfo.state = this.STATE_SUCCESS;
+                  break;
+               }
+               case 1:
+               {
+                  fileInfo.nodeRef = "workspace://SpacesStore/" + response.id;
+                  fileInfo.fileName = response.name;
+                  fileInfo.state = this.STATE_SUCCESS;
+               }
+               default:
```
Comment (MartinDoyleUK) [inline aikau/src/main/resources/alfresco/services/_BaseUploadService.js]: Switch fall-through (will need fixing when adding unit tests)

**Comment by kevinroast** [comment]
>Switch fall-through (will need fixing when adding unit tests)
Yes that is a mistake doh.

**Comment by kevinroast** [comment]
Fixed missing break statement.

**Comment by MartinDoyleUK** [comment]
Superseded by #848.

**Pull request closed without merging**